analyses/molecular-subtyping-integrate/01-integrate-subtyping.Rmd

```diff
@@ -0,0 +1,196 @@
+---
+title: "Integrate molecular subtyping results"
+output: 
+  html_notebook:
+    toc: true
+    toc_float: true
+author: Krutika Gaonkar for D3b
+date: 2020
+params:
+   is_ci: FALSE
+---
+
+The purpose of this notebook is to integrate molecular subtyping results from 
+[molecular-subtyping-pathology](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-pathology) with `pbta-histologies-base.tsv`.
+
+Here we are using v18 `pbta-histologies-base.tsv`
```

Comment:
Just to clarify, (and sorry if this is written elsewhere, but it may be good to add it here too) is the purpose of `pbta-histologies-base.tsv` that you write the subtyping results there and that in the next version of the release, the results in `pbta-histologies-base.tsv` are added to `pbta-histologies.tsv`?

Reply:
Aka it seems like `pbta-histologies-base.tsv` is like the working draft?

Reply:
Right, we can call `pbta-histologies-base.tsv` a working draft. 
In each release there will be a `pbta-histologies-base.tsv` are added to `pbta-histologies.tsv`. The **PBTA Histologies 1-7** PR  use the base histology file which have any integrated_diagnosis,Notes and molecular_subtype as NA through all the molecular-subtyping-XYZ . Through this process we gradually create the `pbta-histologies.tsv` for the same release.

You might have noticed that the modules (other than subtyping) were all using RNA data.
https://github.com/kgaonkar6/OpenPBTA-analysis/blob/760f7a42d253edcaf3341ac0873634e55590e59e/scripts/run-for-subtyping.sh#L13-L32

We only ran these modules because along with the `pbta-histologies-base.tsv` we also needed to update these files with the new RNA samples to use in our molecular subtyping modules.

Reply:
Can you put some form of your description here in the beginning of this notebook?